wandb/sdk/internal/sender.py

```diff
@@ -1586,6 +1597,12 @@ def _send_artifact(
         )
 
         self._job_builder._handle_server_artifact(res, artifact)
+
+        if artifact.manifest.manifest_file:
+            try:
+                os.remove(artifact.manifest.manifest_file)
+            except FileNotFoundError:
+                pass
```

Comment:
nit: feel free to ignore as it only saves a couple lines, but could consider `contextlib.suppress()` if this pattern keeps coming up
```suggestion
            with contextlib.suppress(FileNotFoundError):
                os.remove(artifact.manifest.manifest_file)
```